frontend/server/src/test/java/com/amazonaws/ml/mms/MockWorker.java

```diff
@@ -182,13 +181,13 @@ public void channelRead0(ChannelHandlerContext ctx, BaseModelRequest msg) {
             if (msg instanceof ModelInferenceRequest) {
                 List<RequestBatch> requestBatches = ((ModelInferenceRequest) msg).getRequestBatch();
                 List<Predictions> predictions = new ArrayList<>(requestBatches.size());
-                Base64.Encoder encoder = Base64.getEncoder();
+                //Base64.Encoder encoder = Base64.getEncoder();
                 for (RequestBatch requestBatch : requestBatches) {
                     String requestId = requestBatch.getRequestId();
                     Predictions prediction = new Predictions();
                     prediction.setRequestId(requestId);
-                    prediction.setValue(
-                            encoder.encodeToString("OK".getBytes(StandardCharsets.UTF_8)));
+                    //                    prediction.setValue(
```

Comment:
remove

Reply:
I think we can clean this file once we enable the tests.